src/pages/OfferNew/Compare/CompareTable.tsx

```diff
@@ -310,7 +311,12 @@ export const CompareTable = (props: Props) => {
     <Container>
       <InsurancePropertiesSection>
         <ColumnHead>
-          <SubHeadingBlack>Skydd</SubHeadingBlack>
+          <SubHeadingBlack>
+            <TranslationsConsumer textKey="COMPARE_TABLE_TITLE">
```

Comment:
Not necessarily something we need to do in the context of this PR but just a thought:

It would be super-nice if we could convert this thing from a render props-based API to a hooks based API 🙄

Reply:
Yeah, I know right! Something I've been thinking about too, but I guess that's an issue for later 🙄

Reply:
Yeah! Let's let it stew for a bit in the back of our heads and then do it later